azkaban-web-server/build.gradle

```diff
@@ -50,27 +71,55 @@ sourceSets {
   }
 }
 
-bower {
-  installBase = 'bower' // <1>
+task movingJsTojsToPackage(dependsOn: ['npm_install']) {
 
-  'moment'('2.14.1'){
-    source 'min/moment.min.js' >> '/'
+  doLast {
+    copy {
+      from "node_modules/later/later.min.js"
+      into 'build/jsToPackage'
+    }
+    copy {
```

Comment:
I understand but it totally makes sense to me since the distribution task is the one place holder to gather all that is required in the package. This would avoid having copy tasks in different places.

This may increase the size of the distribution config but that's ok :)